packages/modules/devices/byd/byd/bat.py

```diff
@@ -48,7 +49,7 @@ def get_values(self) -> Tuple[float, float]:
         return BydParser.parse(resp.text)
 
 
-class BydParser(HTMLParser):
+class BydParser(HTMLParser, AbstractBat):
```

Comment:
Ich denkenicht, dass die neue Basisklasse hier hingehört.
```suggestion
class BydParser(HTMLParser):
```

Reply:
umgesetzt